src/docs/product/integrations/issue-tracking/height/index.mdx

```diff
@@ -0,0 +1,53 @@
+---
+title: Height
+sidebar_order: 1
+description: "Learn more about Sentry's Height integration and how you can effortlessly create new Height issues from Sentry and link Sentry issues to existing Height tasks."
+---
+
+Height is designed to work for every type of team: no matter how you work, Height’s workflows are flexible enough to support you. From software engineering to marketing to business operations to design, every kind of team can track their projects and collaborate in one place.
+
+Navigate through your tasks and update them lightning fast with our Command palette and customizable shortcuts. And best of all, with power user features hidden under the hood, Height is simple enough for anyone to get started, regardless of their project management experience.
+
+Our Height Sentry integration allows your team to record and quickly resolve critical errors by linking new or existing Height tasks to Sentry issues. Linking tasks gives your teammates visibility into updates from Sentry, and can automatically mark Sentry issues as Resolved as the task status changes in Height. Additionally, set up custom alerts to create new Height tasks when key triggers you care about, like reported errors or session crashes, spike above a certain threshold.
+
+Height needs to set up only once per organization, then it is available for _all_ projects. It is maintained and supported by the company that created it. For more details, see [Integration Platform](/product/integrations/integration-platform/).
+
+## Install and Configure
+
+<Note>
+
+Sentry owner, manager, or admin permissions are required to install this integration.
+
+Height **won't** work with self-hosted Sentry.
+
+</Note>
+
+1. Navigate to **Settings > Integrations > Height**
+
+   ![](height-install.png)
+
+## Issue Management
+
+Issue tracking allows you to create Height tasks from within Sentry, and link Sentry issues to existing Height tasks. Issue management can be configured in two ways&mdash;automatically or manually.
```

Comment:
We add spaces around our em-dashes (much to my dismay). I've used Shift + Option + dash to create this. It'll look like an en-dash in the diff file, but it should render properly in the preview/production.

```suggestion
Issue tracking allows you to create Height tasks from within Sentry and link Sentry issues to existing Height tasks. Issue management can be configured in two ways — automatically or manually.
```